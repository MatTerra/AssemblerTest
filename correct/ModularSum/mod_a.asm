MOD_A: begin
section data
VAL: CONST 5
section text
Y: EXTERN
INPUT Y
LOAD VAL
ADD Y
STORE Y
JMPP MOD_B
MOD_B: EXTERN
L1: STOP
PUBLIC VAL
PUBLIC L1
EnD
