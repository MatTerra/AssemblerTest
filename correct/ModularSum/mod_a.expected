> 5
PC <- 2
ACC <- 0

PC <- 4
ACC <- 5

PC <- 6
ACC <- 10

PC <- 8
ACC <- 10

PC <- 12
ACC <- 10

PC <- 14
ACC <- 10
OUTPUT: 10

PC <- 16
ACC <- 10
OUTPUT: 5

PC <- 18
ACC <- 10
OUTPUT: 10

PC <- 10
ACC <- 10

PC <- 11
ACC <- 10

